transformerlab/routers/data.py

```diff
@@ -59,28 +59,24 @@ async def dataset_info(dataset_id: str):
 
 
 @router.get("/preview", summary="Preview the contents of a dataset.")
-async def dataset_preview(dataset_id: str):
+async def dataset_preview(dataset_id: str, offset: int = 0, limit: int = 10):
     d = await db.get_dataset(dataset_id)
-
+    datasetLen = 0
     result = []
-
-    print(d)
-
+    #This means it is a custom dataset the user uploaded
```

Comment:
We need a maximum on the limit (and it should be documented in the function docs). So if someone submits a limit of more than, say, 1000, it will only return max 1000 rows regardless